### DISCUSSION TITLE
Where would I put an extra "X-…" HTML header (Docker install)?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6152] [Q&A] [answered]
Say, for some reason, I wanted to put an extra "X-…" header in the HTML headers for AzuraCast public pages. (Assume a "[GNU Terry Pratchett](http://www.gnuterrypratchett.com/)" as an example.)

With the rolling release Docker install, where would I put that?

Is there maybe already an advanced config option I overlooked?

**Comment by Vaalyn** [comment, marked as answer]
I think you'd need to add a [custom nginx config](https://docs.azuracast.com/en/administration/docker#adding-custom-nginx-configurations) to do that.

For reference see the templates for our nginx config: https://github.com/AzuraCast/AzuraCast/tree/main/util/docker/web/nginx

**Comment by Moonbase59** [comment]
@Vaalyn Thanks for the pointers! That should indeed work, and (auto-)including conf’s in `/etc/nginx/azuracast.conf.d/` is a good way, thanks to whoever thought about that!
